ppr-ui/src/composables/useInputRules.ts

```diff
@@ -55,13 +55,23 @@ export const useInputRules = () => {
   }
 
   const isEmail = (): Array<Function> => [
-    (v: string) => !!v || 'Email address is required',
+    (v: string) => !!v || 'Enter an email address',
     (v: string) => {
       const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-      return pattern.test(v) || 'Valid email is required'
+      return pattern.test(v) || 'Enter a valid email address'
     }
   ]
 
+  const isPhone = (
+    maxDigits: number = null,
+    customMsg: string = null
+  ): Array<Function> => {
+    return [
+      v => ((v && maxDigits) ? v.length >= maxDigits : true) || 'Minimum 10 characters',
```

Comment:
We probably don't need this check, as we can use `minLength` instead.

Reply:
The reason I did that is because of v-mask.  It treats the masked characters as as character.   Those characters are not entered by the user, but are included in the field... The mask characters will be stripped when writing to store.  Doug commented on this.. I am going to make that change now.